scripts/build-cdn-indexes.ts

```diff
@@ -0,0 +1,74 @@
+import path from 'path';
+import themes from '../themes.json';
+import fs from 'node:fs';
+import c from 'chalk';
+
+const getSystems = () => {
+  const result: string[][] = [];
+  Object.entries(themes).forEach((theme) => {
+    const config = theme[1];
+    if ('incomplete' in config) return;
+
+    const shortname =
+      config.packageName.split('-')[0] === 'design' ? 'core' : config.packageName.split('-')[1];
+    const packageFile = path.join('packages', config.packageName, 'package.json');
+    const version = JSON.parse(fs.readFileSync(packageFile, 'utf8')).version;
+
+    result.push([config.packageName, version, shortname]);
+  });
+  return result;
+};
+
+getSystems().forEach((sysinfo) => {
+  const [system, version, shortname] = sysinfo;
+
+  const distPath = path.join('packages', system, 'dist');
+  const themeCssPath = `${distPath}/css/${shortname}-theme.css`;
+  const htmlDoc = `<!DOCTYPE html>
+<html lang="en">
+  <meta charset="utf-8" />
+  <head>
+    <meta name=“viewport” content=“width=device-width, initial-scale=1, shrink-to-fit=no”>
+<title>CMSDS CDN Version Index</title>
+    <link rel="stylesheet" href="https://design.cms.gov/cdn/${system}/${version}/css/index.css" />
+    <link rel="stylesheet" href="https://design.cms.gov/cdn/${system}/${version}/css/${shortname}-theme.css" />
+    <script type="text/javascript" src="https://design.cms.gov/cdn/${system}/${version}/js/react.production.min.js"></script>
+    <script type="text/javascript" src="https://design.cms.gov/cdn/${system}/${version}/js/react-dom.production.min.js"></script>
+    <script type="text/javascript" src="https://design.cms.gov/cdn/${system}/${version}/js/bundle.js"></script>
+  </head>
+  <body class="ds-base" style="margin: 0">
+    <div id="usa-banner"></div>
+    <script>
+      ReactDOM.render(
+        React.createElement(DesignSystem.UsaBanner),
+        document.getElementById('usa-banner')
+      );
```

Comment:
Out of scope now, but it would be kind of cool to use the web component version here. I guess it depends on how we end up using these for monitoring.

Reply:
Yeah sure, could always pivot to that, didn't want to include anything 'experimental' :)